ui/hooks/gasFeeInput/useMaxPriorityFeePerGasInput.js

```diff
@@ -7,6 +7,8 @@ import { isLegacyTransaction } from '../../helpers/utils/transactions.util';
 import { hexWEIToDecGWEI } from '../../../shared/modules/conversion.utils';
 import { feeParamsAreCustom, getGasFeeEstimate } from './utils';
 
+const isNullOrUndefined = (value) => value === null || value === undefined;
```

Comment:
Is it worth moving this to a shared utilities file such as `ui/helpers/utils/util.js`?

Reply:
this can be done, LGTM as it's a release blocker